packages/ui/src/Fx/Popover.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode } from 'react';
+
+import { Paper, Text } from '..';
+
+interface Props {
+  children: ReactNode;
+  onClose: () => void;
+}
+
+const Popover = ({ children, onClose, ...rest }: Props): ReactNode => {
```

Comment:
What;s that for? is that a part of FX? please dlete it

Reply:
deleted